registry/consul_registry.go

```diff
@@ -14,6 +14,10 @@ import (
 	hash "github.com/mitchellh/hashstructure"
 )
 
+const (
+	ConsulRegisterTCPCheckKey = "consul_register_tcp_check"
```

Comment:
Do not declare this. We don't need a const for this.

Reply:
Three reasons why I use constant in my code are: (1) minimize typo, e.g. "consul_register_tpc_check"  (2) readability, (3) Easy on refactoring (e.g. changing from "consul_register_tcp_check" to "register_tcp_check" will not effect other files).